core/src/main/java/com/linecorp/armeria/server/AbstractVirtualHostBuilder.java

```diff
@@ -283,75 +281,75 @@ B service(T serviceWithPathMappings, Function<T, R> decorator) {
      * Binds the specified annotated service object under the path prefix {@code "/"}.
      */
     public B annotatedService(Object service) {
-        return annotatedService("/", service);
+        return annotatedService("/", service, ImmutableList.of(), Function.identity());
     }
 
     /**
      * Binds the specified annotated service object under the path prefix {@code "/"}.
```

Comment:
Could you add `@param exceptionHandlersAndConverters` to explain what it has to contain?